duckduckgo/iOS
# Remove Add DuckDuckGo to Safari from Settings

Task/Issue URL: https://app.asana.com/0/856498667320406/1199534840285318
Tech Design URL:
CC:

**Description**:
Remove Add DuckDuckGo to Safari from Settings.

**Steps to test this PR**:
1. Open Settings.
2. Ensure screen works as expected and layout is correct.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
